src/collectives/device/msccl_kernel_impl.h

```diff
@@ -298,9 +305,13 @@ __device__ __forceinline__ void mscclRunInterpreter(
         int thisCount = min(maxAllowedCount, count - c);
         int thisNelem = nelem * thisCount;
         if (t->type == MSCCL_SEND)
```

Comment:
need { ... }

Reply:
My bad..had it in my local version. Just changed.